### ISSUE TITLE
status: ERROR errors: "column out of bounds: 0 [1, 1]"

### ISSUE THREAD

**Issue by EgorBu** [bblfsh/python-driver#57]
Hi,
I tried to extract UAST from python file and got an unexpected result - result of extraction starts with the 
error message.

The content of `exp.py`:
```
# Here you can find the code of /app/exp.py:
# find x in range (-2, 2) that should minimize math.sin function
import math
from hyperopt import fmin, tpe, hp
from hyperopt.mongoexp import MongoTrials

mongodb = "mongodb"
db_port = "27017"
db_name = "ml_exp"
exp_key = "exp1"

trials = MongoTrials("mongo://%s:%s/%s/jobs" % (mongodb, db_port, db_name), exp_key="exp1")
best = fmin(math.sin, hp.uniform("x", -2, 2), trials=trials, algo=tpe.suggest, max_evals=10)
print("Result:", best)
```

and result of extraction:
```
status: ERROR
errors: "column out of bounds: 0 [1, 1]"
uast {
  internal_type: "Module"
...
```
Full UAST is attached: 
[exp.uast.txt](https://github.com/bblfsh/python-driver/files/1208902/exp.uast.txt)

PS:
is it correct that we have several lines of roles?
```
      roles: STRING_LITERAL
      roles: EXPRESSION
      roles: ASSIGNMENT_VALUE
```

**Comment by juanjux** [comment]
Fixed by #59 and bblfsh/python-pydetector#17  when merged.

**Comment by juanjux** [comment]
Note, there are still two instances where this can happen:

1. Empty files because of a [bug in the SDK](bblfsh/sdk#160).

2. Code with code embedded inside f-strings.

In both cases the UAST will be produced and you can ignore the error.
